package.json

```diff
@@ -34,6 +35,7 @@
     "eslint": "8.45.0",
     "eslint-config-prettier": "8.8.0",
     "eslint-plugin-import": "2.27.5",
+    "nodemon": "^3.0.1",
```

Comment:
I never used this library before but looks like a great tool. Can we use exact version for dependencies as a convention?

Reply:
```suggestion
    "nodemon": "3.0.1",
```